src/lib/eslint/BaseEslintEngine.ts

```diff
@@ -35,18 +39,15 @@ export interface EslintStrategy {
 	/** Get engine that strategy supports */
 	getEngine(): ENGINE;
 
-	/** Get eslint engine to use for scanning. */
-	getRunConfig(engineOptions: Map<string, string>): Promise<Record<string, any>>;
+	/** Get config options for eslint engine to use for scanning. */
+	getRunOptions(engineOptions: Map<string, string>): Promise<ESLint.Options>;
```

Comment:
Renamed method to `getRunOptions`, to avoid confusion with the various Config-related types.